opsdroid/testing/fixtures.py

```diff
@@ -76,17 +76,20 @@ def opsdroid() -> OpsDroid:
 
 
 @pytest.fixture
-def mock_api_obj():
+def mock_api_obj(request) -> ExternalAPIMockServer:
     """Initialize instance of :class:`opsdroid.testing.ExternalAPIMockServer`.
 
     This fixture returns an instance that hasn't been started, allowing you to
     add routes to the server before it is started.
     There are a few options of how to start the server once this fixture has
     been used:
 
-        * Use `:meth:opsdroid.testing.ExternalAPIMockServer.run_test`
         * Use the ``mock_api`` fixture in the test *after* this fixture has
           been called to setup a route.
+        * Use :meth:`opsdroid.testing.ExternalAPIMockServer.run_test`
```

Comment:
Should we just remove `run_test` at this point, I personally don't think it's a pattern we should support.

Reply:
If we have the context manager, we probably don't need this one 🤔